test/helpers/JsoupHelpers.scala

```diff
@@ -22,7 +22,7 @@ import org.jsoup.select.Elements
 import play.twirl.api.Html
 
 trait JsoupHelpers {
-  implicit class RichHtml(html: Html) {
+  extension (html: Html) {
```

Comment:
nice

Reply:
Yeah, the `implicit` keyword was so overloaded before... this new syntax is much more explicit 😅